src/test/java/org/broadinstitute/consent/http/service/EmailServiceTest.java

```diff
@@ -106,10 +135,41 @@ public void testSendDataCustodianApprovalMessage() {
         }
     }
 
+    @Test
+    public void testSendNewResearcherEmail() throws Exception {
+        initFakeService();
+        User user = new User();
+        user.setUserId(1234);
+        user.setDisplayName("John Doe");
+
+        User so = new User();
+        user.setEmail("fake_email@asdf.com");
+        try {
+            service.sendNewResearcherMessage(user, so);
+        } catch (Exception e) {
+            fail("Should not fail sending message: " + e);
+        }
+
+        verify(sendGridAPI, times(1)).sendNewResearcherLibraryRequestMessage(any(), any());
+        verify(templateHelper, times(1)).getNewResearcherLibraryRequestTemplate("John Doe", serverUrl);
```

Comment:
I added this in place of checking that the template was right, in addition to the new template test. My main concern was that we don't know if the template was being generated "correctly" with the right arguments... this alleviates that.